index.html

```diff
@@ -11,17 +11,17 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
-    <title>Document</title>
+    <title>BANCUNGUYE's portfolio</title>
 </head>
 <body>
    <header>
       <nav>
           <div class="nav_bar">
              <div class="logo">
-                <a href="#"> Aima.b</a>
+                <a href="#"> <h1 class="logo-h1">Aima.b</h1></a>
```

Comment:
- [ ] The logo is correct in the navbar but the logo is not a `heading` tag. Please take a look [at this article](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) to understand better the hierarchy and the use of the `heading` tags and [this one](https://www.reliablesoft.net/h1-tag/) to understand better the use of the `h1` tag. The page should have one but no more than one `h1` tag. Kindly remove it for better SEO and semantics.